deploy/csi-rclone/values.yaml

```diff
@@ -59,3 +59,4 @@ csiNodepluginRclone:
   tolerations: []
   affinity: {}
 kubernetesClusterDomain: cluster.local
+logLevel: NOTICE
```

Comment:
Do you think we could document here the possible values? `INFO` `DEBUG` `WARN` I don't know if they are valid...